### PR TITLE
Lower the FUZZER value to 37.5 seconds

### PR DESCRIPTION
**Pull request by Navid200**
**Why do we need this?**
This reduces the delta between what is shown on screen and what is in the data table ( what is sent to Nightscout).

A consequence of the delta being great is that occasionally, when the reading arrival time is very close to the boundary of the grid buckets, the readings on the xDrip main screen are not equidistant.  Lowering the value of the FUZZER to 37.5 seconds (from 2.5 minutes) will make the points look equidistant even if they are not.

My hope is that, if everyone is happy with this new setting, some day, we will remove the FUZZER altogether.  So, this is a first step.

Fixes https://github.com/NightscoutFoundation/xDrip/issues/1537  

**Is there a workaround?**
No  

**Are there any side-effects to this change?**
Unless the user enables the setting, nothing changes from the user perspective.  

**Tests**  
This has been tested on an Android 9 handset.

### CODE REVIEW

**Comment by jamorham** [comment]
As there are quite a few lines here, could you summarize the changes that you understand to be in this PR? thanks

**Comment by Navid200** [comment]
The following is a list of lines that contain real changes (in the BgGraphBuilder file).  The rest are auto formatting.

BgGraphBuilder.java:  85, 458, 1431
